website/docs/reference/warehouse-profiles/spark-profile.md

```diff
@@ -86,6 +89,7 @@ your_profile_name:
       user: [user]
       auth: [e.g. KERBEROS]
       kerberos_service_name: [e.g. hive]
+      use_ssl: [true|false]   # value of hive.server2.use.SSL, default false
```

Comment:
perhaps this would be better bundled into a `connection_parameters` config dict? idk...
https://github.com/databricks/dbt-databricks/blob/13011d064ffd0fba91e9542543595b815d0b8510/dbt/adapters/databricks/connections.py#L58